docs/user/getting-started.md

```diff
@@ -24,11 +24,33 @@ The above commands will deploy Kubeapps into the `kubeapps` namespace in your cl
 
 Access to the Dashboard requires a Kubernetes API token to authenticate with the Kubernetes API server.
 
-```
+For Linux:
+
+```bash
```

Comment:
You need to move the following two commands outside the platform specific instructions right?

```bash
kubectl create serviceaccount kubeapps-operator
kubectl create clusterrolebinding kubeapps-operator --clusterrole=cluster-admin --serviceaccount=default:kubeapps-operator 
```